### ISSUE TITLE
feature reuqest: jump the cursor in editor when ctrl+click in preview

### ISSUE THREAD

**Issue by ZhuangQu** [shd101wyy/vscode-markdown-preview-enhanced#588]
In the default markdown preview of vs code, there is a feature: when we `double click` the mouse in the preview, the cursor in the editor will automatically jump to the corresponding position. This feature is very practical, which is helpful for us to quickly locate and modify content. The PDF preview of `latex workshop` also has this feature, which used by `ctrl+click`. This plug-in also needs this feature.

**Comment by ZhuangQu** [comment]
This issue has been published in [there](https://github.com/shd101wyy/markdown-preview-enhanced/issues/1691) so I close it.

**Comment by eizanprime** [comment]
Is this feature implemented yet ? This extension has many very useful features, but the preview itself is synchronized better in the native vscode version which is a bit sad..

**Comment by iammrallblue** [comment]
> Is this feature implemented yet? This extension has many very useful features, but the preview itself is synchronized better in the native vscode version which is a bit sad...

Nope, The most recent version which is 0.6.8 still has not implemented this feature, I am really expecting that developer can add it to the next version. @shd101wyy can you please update this "click-jump" feature?

**Comment by iammrallblue** [comment]
@shd101wyy 
<img width="365" alt="image" src="https://user-images.githubusercontent.com/504961/228408299-a2fb7412-0602-4db6-8800-408c6fa674e8.png">
please consider it, this is a really good feature, I wish you are going to update it to the next version.  very grateful sir.

**Comment by ZhuangQu** [comment]
Do the author encounter any technical difficulties with this feature
